<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1">
   <item>
    <widget class="QFrame" name="left_frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <layout class="QVBoxLayout" name="left_vLayout"/>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="center_vLayout">
     <property name="spacing">
      <number>16</number>
     </property>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="left_addWidgetBtn">
       <property name="text">
        <string>&lt;&lt;&lt; Add Widget</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="left_removeWidgetBtn">
       <property name="text">
        <string>&lt;&lt;&lt; Remove Widget</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="right_addWidgetBtn">
       <property name="text">
        <string>Add Widget &gt;&gt;&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="right_removeWidgetBtn">
       <property name="text">
        <string>Remove Widget &gt;&gt;&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="right_frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <layout class="QVBoxLayout" name="right_vLayout"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
